lib/stripe/stripe_object.rb

```diff
@@ -221,9 +225,9 @@ def add_accessors(keys, values)
         keys.each do |k|
           next if f.include?(k)
           next if @@permanent_attributes.include?(k)
-          k_eq = :"#{k}="
+
           define_method(k) { @values[k] }
-          define_method(k_eq) do |v|
+          define_method(:"#{k}=") do |v|
```

Comment:
This is kind of a minor stylistic change for consistency: we don't set a separate variable below for the question mark accessor, so I removed `k_eq` here.